docs/book/features/block_volume.md

```diff
@@ -375,3 +376,67 @@ Cluster
             VolumeAttributes:      type=vSphere CNS Block Volume
         Events:                <none>
     ```
+
+### Statically Provision a Block Volume in Tanzu Kubernetes Grid Cluster<a id="static_volume_provisioning_tkg"></a>
+
+You can statically create a block volume in a Tanzu Kubernetes Grid (TKG) Cluster using an unused PersistentVolumeClaim (PVC) from the Project Pacific cluster only if the latter satisfies the following conditions:
```

Comment:
`unused PersistentVolumeClaim (PVC) from the Project Pacific cluster` => `unused PersistentVolumeClaim (PVC) from a Supervisor cluster namespace`

Reply:
done